### ISSUE TITLE
2.0.1 setup.py dependencies breaking app start-up?

### ISSUE THREAD

**Issue by hartwork** [slgobinath/SafeEyes#224]
Hi!

I haven't fully figured this out myself yet, maybe you can help me get a full picture. The symptom is that the changes from bf57dbd72f2be30dfc78bd8ea80bf568620793d8 add pip dependencies that cause trouble during start-up (despite the related packages being installed):

```console
# safeeyes
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.5/safeeyes", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 3161, in <module>
    @_call_aside
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 3145, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 3174, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 666, in _build_master
    ws.require(__requires__)
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 984, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 870, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'gi' distribution was not found and is required by safeeyes

# python3.5 -c 'import gi; print(gi.__file__)'
/usr/lib64/python3.5/site-packages/gi/__init__.py
```

For `gi` it seems that the correct name to use would be `PyGObject` (https://pypi.python.org/pypi/PyGObject). For dbus, I'm clueless still. Neither `dbus` or `dbus-python` (https://pypi.python.org/pypi/dbus-python) seem to work for a name, verified with `pkg_resources.get_distribution`. The system-wide installation of dbus-python does not seem to contain an "egg", maybe that's related.

I use [safeeyes-2.0.1-setup-py.patch](https://github.com/gentoo/gentoo/blob/00ddbcbbeec0caa0aa7b04788baf793da46997a1/x11-misc/safeeyes/files/safeeyes-2.0.1-setup-py.patch) for a workaround in Gentoo right now, but I have a feeling we need something better to apply for everyone.

Any ideas?

**Comment by slgobinath** [comment]
Hi,
I figured out the same issue only after releasing the version so I stopped releasing them to Ubuntu and Arch.
You are right the problem is with those dependencies and I +1 your patch. Since those dependencies were added by the Debian maintainer I am afraid whether those have any impacts on the Debian build.

Once I get a confirmation from him, I will apply the fix and release to other versions. For the moment please go ahead with your patch.

Thanks

Related issue: https://github.com/slgobinath/SafeEyes/issues/223

**Comment by slgobinath** [comment]
Since there is no update on it, I have applied your patch and released the version 2.0.2 (Since 2.0.1 was uploaded to PyPi I increased the version). The new version includes the fix and improved Dutch translation.

Thanks

**Comment by hartwork** [comment]
Very nice, thank you!
